main.go

```diff
@@ -161,6 +153,31 @@ func scytale(arguments []string) int {
 	return 0
 }
 
+func loadTracing(v *viper.Viper, appName string) (candlelight.Tracing, error) {
+	var tracing = candlelight.Tracing{
+		Enabled:        false,
+		Propagator:     propagation.TraceContext{},
+		TracerProvider: trace.NewNoopTracerProvider(),
+	}
+	if v.IsSet(tracingConfigKey) {
+		var traceConfig candlelight.Config
+		err := v.UnmarshalKey(tracingConfigKey, &traceConfig)
+		if err != nil {
+			return candlelight.Tracing{}, err
+		}
+		traceConfig.ApplicationName = appName
+		tracerProvider, err := candlelight.ConfigureTracerProvider(traceConfig)
+		if err != nil {
+			return candlelight.Tracing{}, err
+		}
+		if len(traceConfig.Provider) != 0 && traceConfig.Provider != candlelight.DefaultTracerProvider {
```

Comment:
Isn't the `DefaultTracerProvider` an actual tracer? Or is it a No-op?  If it's a tracing provider, then tracing should be `Enabled` in that case, right?

Reply:
Nevermind, looks like the default is no-op, so this is fine. 👍